src/test/java/ezvcard/io/roundtrip/XCardRoundTripTest.java

```diff
@@ -0,0 +1,82 @@
+package ezvcard.io.roundtrip;
+
+import java.io.*;
+
+import org.junit.Test;
+
+import ezvcard.VCardVersion;
+import ezvcard.io.StreamReader;
+import ezvcard.io.StreamWriter;
+import ezvcard.io.xml.XCardReader;
+import ezvcard.io.xml.XCardWriter;
+
+public class XCardRoundTripTest extends RoundTripTestBase {
+
+	public XCardRoundTripTest() throws Exception {
+		updateSamples(VCardVersion.V4_0,
+				"outlook-2003" // &#12; in fburl is not valid xml
+		);
+		updateSamples(VCardVersion.V3_0);
+	}
+
+	@Test
+	public void equals_compare_vcard_4_to_xcard() throws Exception {
+		convertAllFromVCard(VCardVersion.V4_0, true, false,
+				"outlook", // newlines not preserved on linux
```

Comment:
`newlines not preserved on linux`

If I understand this issue correctly, this isn't an issue about newline preservation or operating systems, but about how the lines in the vCard file are **folded**.  The "outlook-2007.vcf" file is laid out so that lines are folded when an encoded newline is encountered.  However, when ez-vcard writes the vCard, it folds the lines when the line gets to be 75 characters long.  The properties look different in the data file, but all data is preserved.  Is this the issue or am I misunderstanding it?